gangplank/cosa/build.go

```diff
@@ -81,6 +89,30 @@ func ReadBuild(dir, buildID, arch string) (*Build, string, error) {
 	}
 
 	b, err := buildParser(f)
+	if err != nil {
+		return b, p, err
+	}
+
+	// if delaydMetaMerge is set, then we need to load up and merge and meta.*.json
+	if b != nil && b.CosaDelayedMetaMerge {
+		log.Info("Searching for extra meta.json files")
+		err = filepath.Walk(p, func(path string, fi os.FileInfo, _ error) error {
```

Comment:
Why [`Walk`](https://golang.org/pkg/path/filepath/#Walk) and not [`WalkDir`](https://golang.org/pkg/path/filepath/#WalkDir)

Reply:
Habit? Updating coming in shortly.

Reply:
@ashcrow `WalkDir` is new to Golang 1.16. F33 has Golang 1.15, so we can't use it get.